### PR TITLE
Add llama_cpp_quickstart.zh-CN.md

### PR DESCRIPTION
**Pull request by joan726**
Add language switching in llama_cpp_quickstart.md.

### CODE REVIEW

**Comment by jason-dai** [comment]
Also update links in https://github.com/intel-analytics/ipex-llm/blob/main/README.zh-CN.md

**Comment by joan726** [comment]
> Also update links in https://github.com/intel-analytics/ipex-llm/blob/main/README.zh-CN.md

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+## 快速入门
+本快速入门指南将引导你完成安装和使用 `ipex-llm` 运行 `llama.cpp`。
+
+### 0 系统环境准备
+IPEX-LLM 现在已支持在 Linux 和 Windows 系统上运行 `llama.cpp`。
+
+#### Linux
+对于 Linux 系统，我们推荐使用 Ubuntu 20.04 或更高版本 (优先推荐 Ubuntu 22.04)。
+
+请仔细参阅网页[在配有 Intel GPU 的 Linux 系统下安装 IPEX-LLM](./install_linux_gpu.md), 首先按照 [Intel GPU 驱动程序安装](./install_linux_gpu.md#install-gpu-driver)步骤安装 Intel GPU 驱动程序，然后参考 [oneAPI 安装](./install_linux_gpu.md#install-oneapi)步骤安装 Intel® oneAPI Base Toolkit 2024.0，以确保后续 IPEX-LLM 能顺利安装。
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:45]: 请仔细参阅网页[在配有 Intel GPU 的 Linux 系统下安装 IPEX-LLM](./install_linux_gpu.md), 首先按照 [Intel GPU 驱动程序安装](./install_linux_gpu.md#install-gpu-driver)步骤安装 Intel GPU 驱动程序，然后参考 [oneAPI 安装](./install_linux_gpu.md#install-oneapi)步骤安装 Intel® oneAPI Base Toolkit 2024.0。

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+- **Linux 用户**:
+  
+  ```bash
+  ./llama-cli -m mistral-7b-instruct-v0.1.Q4_K_M.gguf -n 32 --prompt "Once upon a time, there existed a little girl who liked to have adventures. She wanted to go to places and meet new people, and have fun" -c 1024 -t 8 -e -ngl 99 --color
+  ```
+
+  > **Note**:
+  >
+  > 可以使用 `./main -h` 查看每个参数的详细含义。
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:166]: `./llama-cli -h`

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md [hunk 3]
```diff
@@ -0,0 +1,369 @@
...
+
+  请在 Miniforge Prompt 中运行以下命令。
+
+  ```cmd
+  llama-cli -m mistral-7b-instruct-v0.1.Q4_K_M.gguf -n 32 --prompt "Once upon a time, there existed a little girl who liked to have adventures. She wanted to go to places and meet new people, and have fun" -c 1024 -t 8 -e -ngl 99 --color
+  ```
+
+  > **Note**:
+  >
+  > 可以使用 `main -h` 查看每个参数的详细含义。
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:178]: `llama-cli -h`

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md [hunk 4]
```diff
@@ -0,0 +1,369 @@
...
+llama_print_timings:        eval time =    xx.xx ms /    31 runs   (   xx.xx ms per token,  xx.xx tokens per second)
+llama_print_timings:       total time =    xx.xx ms /    62 tokens
+Log end
+
+```
+
+### 故障排除
+
+#### 1. 无法运行初始化脚本
+如果你无法运行 `init-llama-cpp.bat`, 请确保在你的 conda 环境中已经安装了 `ipex-llm[cpp]`。如果你已安装, 请检查是否已激活正确的 conda 环境。此外，如果你使用的是 Windows，请确保是在提示终端中以管理员权限运行该脚本.
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:318]: 此外，如果你使用的是 Windows，请确保是在提示终端中以管理员权限运行该脚本。

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md [hunk 5]
```diff
@@ -0,0 +1,369 @@
...
+有关如何执行此操作的详细说明，请参阅[此问题](https://github.com/intel-analytics/ipex-llm/issues/10989#issuecomment-2105600469)。
+
+#### 10. sycl7.dll 未找到错误
+如果你在 Linux 或者 Windows 上遇到类似 `System Error: sycl7.dll not found` 的错误, 请根据操作系统进行下列检查:
+
+1. Windows：是否已经安装了 conda 并激活了正确的 conda 环境，环境中是否已经使用 pip 安装了 oneAPI 依赖项
+2. Linux：是否已经在运行 llama.cpp 命令前执行了 `source /opt/intel/oneapi/setvars.sh`。执行此 source 命令只在当前会话有效。
+
+#### 11. 遇到输出乱码请先检查驱动
+如果你遇到输出乱码，请检查 CPU 驱动版本是否 >= [31.0.101.5522](https://www.intel.cn/content/www/cn/zh/download/785597/823163/intel-arc-iris-xe-graphics-windows.html)。如果不是，请参照[这里](./install_linux_gpu.md#install-gpu-driver) 的说明更新你的 GPU 驱动。
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:357]: 如果你遇到输出乱码，请检查 GPU 驱动版本是否 >= [31.0.101.5522](https://www.intel.cn/content/www/cn/zh/download/785597/823163/intel-arc-iris-xe-graphics-windows.html)。

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md (hunk 2 above)
Comment (rnwang04) [reply docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:166]: `main -h` is out-of-date, maybe also help update this part of English version .

**Review by rnwang04 — COMMENTED** [review]
others LGTM

File: docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md (hunk 3 above)
Comment (rnwang04) [reply docs/mddocs/Quickstart/llama_cpp_quickstart.zh-CN.md:178]: `main -h` is out-of-date, maybe also help update this part of English version .

**Review by rnwang04 — APPROVED** [review]
LGTM, please merge it after resolving the conflicts.

**Pull request merged**